tv_apps/smart-system/js/preview_window.js

```diff
@@ -25,7 +25,7 @@
       this.container = configs.rearWindow;
     }
```

Comment:
Although it's not related with this patch..
`This window is inherit the AppWindow` -> `This window inherits AppWindow`